packages/babel-preset-expo/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 
 ### 🐛 Bug fixes
 
+- Fix React compiler support. ([#32187](https://github.com/expo/expo/pull/32187) by [@EvanBacon](https://github.com/EvanBacon))
```

Comment:
Shouldn't this be a major version change?

Reply:
support was never official and it's all still in beta.